pkg/domain/infra/abi/system.go

```diff
@@ -88,3 +99,146 @@ func (ic *ContainerEngine) VarlinkService(_ context.Context, opts entities.Servi
 	}
 	return nil
 }
+
+func (ic *ContainerEngine) SetupRootless(cmd *cobra.Command) error {
```

Comment:
I'd like an ack from @giuseppe about the rootless configuration functions here, even if they aren't wired up

Reply:
Next PR will need his review